gogoogledm.go

```diff
@@ -158,10 +159,22 @@ func (api *DistanceMatrixAPI) sendRequest(origins []Coordinates, destinations []
 		return nil, err
 	}
 
-	resp, err := http.Get(url)
+	client := http.Client{
+		Timeout: 500 * time.Millisecond,
+	}
+
+	var resp *http.Response
+
+	for retries := 0; retries < maxRetries; retries++ {
```

Comment:
As far as I get the code, you will retry even if there is no error...

Reply:
Fixed, missed a commit here 🙈